Terminal.Gui/Views/Button.cs

```diff
@@ -283,7 +283,7 @@ public override bool MouseEvent (MouseEvent me)
 					if (!HasFocus) {
 						SetFocus ();
 						SetNeedsDisplay ();
-						Redraw (Bounds);
+						Draw ();
```

Comment:
No, the intention here is for the `Button` draw first before run the action, instead of only redraw after exit from the action, ti have visual effect immediately.